core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/azureModels.scala

```diff
@@ -33,7 +33,10 @@ final case class AKSCluster(name: String, tags: Map[String, Boolean]) {
 
 final case class WsmManagedAzureIdentity(wsmResourceName: String, managedIdentityName: String)
 
-final case class WsmControlledDatabaseResource(wsmDatabaseName: String, azureDatabaseName: String)
+final case class WsmControlledDatabaseResource(wsmDatabaseName: String,
+                                               azureDatabaseName: String,
+                                               controlledResourceId: Option[UUID] = None
```

Comment:
Why make this an `Option`? I would think the resource should always have an `id` defined?

(Flagged this because later on I see `controlledResourceId.get`, so it seems like we always expect this to be defined.)